lib/Mojo/DOM.pm

```diff
@@ -166,6 +166,18 @@ sub tag {
   return $self;
 }
 
+sub target {
+  my ($self, $submit) = @_;
+  return () if ($self->tag // '') ne 'form';
+  return () unless defined($submit = $self->at($submit || ''));
+  return () if $submit->matches('[disabled]');
+  my $method = uc($submit->attr('formmethod') || $self->attr('method') || 'GET');
```

Comment:
Looking at [the form tag attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#Attributes), I don't see such like `formmethod` or `formaction`. What is the rationale behind this check?

Reply:
See the docs for [button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button)